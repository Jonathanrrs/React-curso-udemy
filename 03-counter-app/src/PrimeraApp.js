// import React from 'react';
// import React, { Fragment } from 'react';

/* Funcional components */

const PrimeraApp = (props) => {
    console.log(props);
    const saludo = 'Hola mundo';

    return (
        <> {/* fragment */}
            <h1>{saludo}</h1>
            {/* <pre>{JSON.stringify(saludo, null, 3)}</pre> */}
            <p>Mi primera aplicaci√≥n</p>
        </>
    );
    
}

export default PrimeraApp;