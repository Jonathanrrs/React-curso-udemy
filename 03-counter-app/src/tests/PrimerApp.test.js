import React from 'react';
import { render } from '@testing-library/react';
import PrimeraApp from '../PrimeraApp';

describe('Pruebas en PrimeraAPP', () => {

    test('Demostrar el mensaje "Hola, soy gokú', () => {
        const saludo = 'Hola, soy Gokú';

        const {getByText} = render( <PrimeraApp saludo={saludo} />);

        expect(getByText(saludo)).toBeInTheDocument();
    }) 
})