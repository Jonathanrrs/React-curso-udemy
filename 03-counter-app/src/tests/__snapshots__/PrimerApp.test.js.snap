// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en PrimeraAPP Debe mostrar PrimerApp correctamente 1`] = `
<Fragment>
  <h1>
    Hola, soy gok√∫
  </h1>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
